feature-group-definitions/appearance.yml

```
@@ -0,0 +1,11 @@
+name: appearance
+description: "The `appearance` CSS property controls the appearance of form controls. Using `appearance: none` disables any default native appearance and allows the elements to be styled with CSS like any other element."
```

@zcorpan is this a reasonable description of `appearance` for web developers? I'm focusing on the main use case and leaving aside some of the bizarre things that can be done.

---

Generally reasonable but "like any other element" is not accurate because some form controls still have weird behavior, e.g. buttons.
```suggestion
description: "The `appearance` CSS property controls the appearance of form controls. Using `appearance: none` disables any default native appearance and allows the elements to be styled with CSS."
```